modules/use-process-baselines.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_module-type: CONCEPT
+[id="use-process-baselines_{context}"]
+= Using process baselines
+
+[role="_abstract"]
+You can minimize risk by using process baselining for infrastructure security. With this approach, {product-title} first discovers existing processes and creates a baseline.
+Then, it operates in the default deny-all mode and only allows processes listed in the baseline to run.
```

Comment:
I don't think you need a comma here ("Then,") to be consistent with line 17 below.